packages/@aws-cdk/core/lib/custom-resource-provider/cross-region-export-providers/cross-region-ssm-writer-handler/index.ts

```diff
@@ -61,6 +57,26 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
   }
 };
 
+function chunkArray<T>(arr: T[], size: number): T[][] {
+  return arr.length > size
+    ? [arr.slice(0, size), ...chunkArray(arr.slice(size), size)]
+    : [arr];
+}
+
+async function deleteParameters(ssm: SSM, names: string[]): Promise<void> {
```

Comment:
Can you add a comment here explaining why we are splitting it into groups of 10?